sfs/time/source.py

```diff
@@ -4,6 +4,23 @@
 
 .. include:: math-definitions.rst
 
+.. plot::
+    :context: reset
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import sfs
+
+    plt.rcParams['figure.figsize'] = 6, 6  # inches
+
+    xs = 1.5, 1, 0  # point source
+    t = np.linalg.norm(xs) / sfs.default.c
+
+    # Impulsive excitation
+    signal = sfs.util.DelayedSignal(np.insert(np.zeros(511), 0, 1), 44100, 0)
```

Comment:
Same as above.

Reply:
I took the liberty of using `unit_impulse()` in bf104a3dae8b1ff44cf91fa8bdd3010daff2b19f.